crates/sui-transactional-test-runner/src/test_adapter.rs

```diff
@@ -1039,6 +1017,73 @@ impl<'a> MoveTestAdapter<'a> for SuiTestAdapter<'a> {
 
                 Ok(None)
             }
+            SuiSubcommand::Bench(
+                RunCommand {
+                    signers,
+                    args,
+                    type_args,
+                    gas_budget,
+                    syntax,
+                    name,
+                },
+                extra_args,
+            ) => {
+                let (raw_addr, module_name, name) = name.unwrap();
+
+                assert!(
+                    syntax.is_none(),
+                    "syntax flag meaningless with function execution"
+                );
+
+                let addr = self.compiled_state().resolve_address(&raw_addr);
+                let module_id = ModuleId::new(addr, module_name);
+                let type_args = self.compiled_state().resolve_type_args(type_args)?;
+                let args = self.compiled_state().resolve_args(args)?;
+
+                let tx = self
+                    .build_function_call_tx(
+                        &module_id,
+                        name.as_ident_str(),
+                        type_args.clone(),
+                        signers.clone(),
+                        args.clone(),
+                        gas_budget,
+                        extra_args.clone(),
+                    )
+                    .unwrap();
+
+                let objects = self.executor.read_objects(tx.clone()).await?;
+                self.next_task();
```

Comment:
Why is this in the middle of the task instead of at the top?

Reply:
A better question is, why do i have this at all, since `call_function()` calls this